### PR TITLE
GCW-2366 Add staff note to order

### PR DESCRIPTION
**Pull request by patrixr**
This is field is meant to be used for the sticky note on the logisitics tab

As seen here: https://xd.adobe.com/view/224f7314-f1c0-40b6-7d7a-929e616be6bb-547e/screen/2f427e12-2183-4a74-869b-eb3d62c8d5e8/iPhone-6-7-8-97/

### CODE REVIEW

File: app/controllers/api/v1/orders_controller.rb [hunk 1]
```diff
@@ -160,15 +161,16 @@ def bool_param(key, default)
...
           :detail_id, :detail_type, :description,
           :state, :cancellation_reason, :state_event,
           :stockit_organisation_id, :stockit_activity_id,
           :people_helped, :beneficiary_id, :booking_type_id, :purpose_description,
-          :address_id,:submitted_by_id, purpose_ids: [], cart_package_ids: [],
+          :address_id,:submitted_by_id, :staff_note,
+          purpose_ids: [], cart_package_ids: [],
           beneficiary_attributes: beneficiary_attributes,
-          address_attributes: address_attributes
+          address_attributes: address_attributes,
```
Comment (abhinav098) [inline app/controllers/api/v1/orders_controller.rb]: @patrixr , unnecessary comma here.

Comment (patrixr) [reply app/controllers/api/v1/orders_controller.rb]: fixed

File: spec/controllers/api/v1/holidays_controller_spec.rb [hunk 2]
```diff
@@ -34,7 +34,6 @@
       it "Should not include current day if it is a holiday", :show_in_doc do
         create(:holiday, holiday: Time.now.beginning_of_day)
         get :available_dates, schedule_days: 6
-        pp response.body
```
Comment (steveyken) [inline spec/controllers/api/v1/holidays_controller_spec.rb]: yay! thanks for finding that.. I'd been searching for 'puts' and 'print' etc etc but couldn't locate it!

Comment (patrixr) [reply spec/controllers/api/v1/holidays_controller_spec.rb]: I could take credit for removing it, but i'm also the one who added it 🤣

**Pull request merged**